components/suggest/src/rs.rs

```diff
@@ -94,6 +104,18 @@ pub(crate) enum SuggestRecord {
     Yelp,
 }
 
+impl SuggestRecord {
+    pub fn get_rs_type(&self) -> &str {
```

Comment:
I like the way you structured this helper! I think it can move to `SuggestionProvider`—and remember that `Amp` and `Wikipedia` need to fetch two types of records, `AmpWikipedia` and `Icon`.

Reply:
Ok sure! I'll update this.